### PR TITLE
CRN-1296 Calendar Page for Working Groups

### PR DESCRIPTION
**Pull request by gabiayako**
Jira ticket: https://asaphub.atlassian.net/browse/CRN-1296

---

## Validation

Hub: https://2576.hub.asap.science/network/working-groups/0182aef4-0d46-4ec7-aae0-584d0b0e3bad/calendar

#### Calendar
calendar-1
calendar-2

### CODE REVIEW

File: apps/crn-frontend/src/network/groups/GroupProfile.tsx [hunk 1]
```diff
@@ -68,7 +68,9 @@ const GroupProfile: FC<GroupProfileProps> = ({ currentTime }) => {
           About={() => (
             <About group={group} groupTeamsElementId={groupTeamsElementId} />
           )}
-          Calendar={() => <Calendar calendars={group.calendars} />}
+          Calendar={() => (
```
Comment (kmaid) [inline apps/crn-frontend/src/network/groups/GroupProfile.tsx:71]: Why is this a function?

File: packages/model/src/working-group.ts [hunk 2]
```diff
@@ -58,10 +58,7 @@ export type WorkingGroupDataObject = {
   pointOfContact?: WorkingGroupLeader;
   complete: boolean;
   shortText: string;
-  calendars: Pick<
-    CalendarDataObject,
-    'id' | 'name' | 'color' | 'groups' | 'workingGroups'
-  >[];
+  calendars: CalendarResponse[];
```
Comment (kmaid) [inline packages/model/src/working-group.ts:61]: 👍

**Review by kmaid — APPROVED** [review]

File: apps/crn-frontend/src/network/groups/GroupProfile.tsx (hunk 1 above)
Comment (gabiayako) [reply apps/crn-frontend/src/network/groups/GroupProfile.tsx:71]: I've made this way so ProfileSwitch can receive `Calendar` directly without having to receive the props Calendar receive :)

File: apps/crn-frontend/src/network/groups/GroupProfile.tsx [hunk 3]
```diff
@@ -68,7 +68,9 @@ const GroupProfile: FC<GroupProfileProps> = ({ currentTime }) => {
...
+            <Calendar calendars={group.calendars} groupType="interest" />
+          )}
```
Comment (kmaid) [inline apps/crn-frontend/src/network/groups/GroupProfile.tsx:73]:
```suggestion
          Calendar={<Calendar calendars={group.calendars} groupType="interest" />}
```

Won't this work?

Comment (gabiayako) [reply apps/crn-frontend/src/network/groups/GroupProfile.tsx:73]: Apparently it doesn't. It appears like this

Screenshot 2023-02-14 at 09 47 32

**Comment by gabiayako** [comment]
force pushing because I rebased to master to solve conflicts

**Pull request merged**